charts/apiclarity/templates/clusterrole.yaml

```diff
@@ -7,8 +7,14 @@ metadata:
     {{ include "apiclarity.labels" . }}
 rules:
 - apiGroups: [""]
-  resources: ["nodes", "services", "pods"]
+  resources: ["nodes", "services", "pods", "configmaps"]
+  verbs: ["get", "list", "watch", "create"]
+- apiGroups: [""]
+  resources: ["configmaps"]
   verbs: ["get", "list", "watch"]
```

Comment:
why `configmaps` duplication and why we need `create` verb?

Reply:
My bad...
I wanted to separate "configmaps" and "secret" from the "global" one as we don't need all the permissions, but I forgot to remove "configmaps" and "create" from the global line.